app/assets/javascripts/application.js.erb

```diff
@@ -110,6 +110,7 @@ $(document).ready( function() {
             row.child( format(row.data()) ).show();
             tr.addClass('shown');
             $.getJSON(APP_PATH + '/json?pbsid='+row.data().pbsid+'&host='+row.data().cluster , function(data) {
+              console.log(row.data());
               console.log(data);
```

Comment:
I would remove the console.log or comment them out since they are useful for debugging but not desirable for production. Or if you want to get fancy you can have these run only if a query param "debug" is set or exists.